src/Transports.AspNetCore/WebSockets/IOperationMessageProcessor.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Transport;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets;
+
+/// <summary>
+/// Processes a stream of messages received from a WebSockets client.
+/// <see cref="IDisposable.Dispose"/> must be called when the connection terminates.
```

Comment:
Do you mean IWebSocketConnection? If so I would add comment that _usually IWebSocketConnection implementation disposes this object_.

Reply:
No the owner of the `IOperationMessageProcessor`, which will be the `IWebSocketHandler` implementation, will dispose of the objects it creates.

See https://github.com/graphql-dotnet/server/pull/812/files#diff-3c1de1415b9d3b97edb7c74eed76976a357d80372e264ef41721a0c4fb297fd5R102-R104

Reply:
Actually, I do also call `Dispose` proactively from the `IWebSocketConnection` implementation in order to prevent additional subscription notifications arriving after the WebSocket is negotiating a graceful shutdown.  It is still necessary in `WebSocketHandler` in case a graceful shutdown does not occur, of course.

https://github.com/graphql-dotnet/server/pull/809/files#diff-9bfc1e96fa77f9fb55180ef1c03bd235d0d755c0154382b67f8436daf4422acbR102-R103

Reply:
```suggestion
/// <see cref="IDisposable.Dispose"/> must be called when the WebSocket connection
/// has received a close message, and/or when the connection terminates.
```

Reply:
Please review again, and provide a suggestion if you think further xml comment changes are necessary.

Reply:
Now I understand.